toolchain/check/import_cpp.cpp

```diff
@@ -500,63 +500,73 @@ static auto IsDeclInjectedClassName(const Context& context,
           ast_context.getRecordType(scope_record_decl)) ==
       ast_context.getCanonicalType(ast_context.getRecordType(record_decl)));
 
-  CARBON_CHECK(name_id ==
-               context.sem_ir()
-                   .classes()
-                   .Get(context.sem_ir()
-                            .insts()
-                            .GetAs<SemIR::ClassDecl>(clang_decl.inst_id)
-                            .class_id)
-                   .name_id);
+  SemIR::ClassId class_id = context.sem_ir()
+                                .insts()
+                                .GetAs<SemIR::ClassDecl>(clang_decl.inst_id)
+                                .class_id;
+  CARBON_CHECK(name_id == context.sem_ir().classes().Get(class_id).name_id);
```

Comment:
```suggestion
  auto class_decl =
      context.sem_ir().insts().GetAs<SemIR::ClassDecl>(clang_decl.inst_id);
  CARBON_CHECK(name_id ==
               context.sem_ir().classes().Get(class_decl.class_id).name_id);
```

Small suggestion here -- the way clang-format wraps is at `.` on an expression after it's too long. But changing the local slightly, you should be able to affect wrapping.

Reply:
Done.